guides/common/modules/proc_configuring-rhui-alternate-content-sources.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Configuring_RHUI_Alternate_Content_Sources_{context}"]
+= Configuring RHUI Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *RHUI*.
+. Generate RHUI certificates using the command provided in the {ProjectWebUI}.
+Ensure that you pass the repo labels of the desired repositories.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the the *Description* field, provide a description for the ACS.
+. Select {SmartProxies} to which the alternate content source is to be synced.
```

Comment:
Also: `. Optional: Select 'Use HTTP proxies' if you want the ACS to use the {SmartProxy}'s HTTP proxy.`